app/presenters/hyrax/collection_presenter.rb

```diff
@@ -25,7 +25,7 @@ def initialize(solr_document, current_ability, request = nil)
     delegate(*Hyrax::CollectionType.collection_type_settings_methods, to: :collection_type, prefix: :collection_type_is)
 
     def collection_type
-      gid = Array.wrap(solr_document.fetch('collection_type_gid_ssim', [Hyrax::CollectionType.find_or_create_default_collection_type.gid])).first
+      gid = Array.wrap(solr_document['collection_type_gid_ssim']).first
```

Comment:
Should this be defined as a property on the `SolrDocument` class so we can do `gid = solr_document.collection_type_gid`?